tensorflow/compiler/xla/g3doc/operation_semantics.md

```diff
@@ -1450,6 +1453,11 @@ If `indices_are_sorted` is set to true then XLA can assume that `start_indices`
 are sorted (in ascending `start_index_map` order) by the user. If they are not
 then the semantics is implementation defined.
 
+If `unique_indices` is set to true then XLA can assume that all
+element scattered to are unique.  So XLA could use non-atomic
+operation. If they are not, then the semantics is implementation
```

Comment:
Let's be a bit clearer: if `unique_indices` is set to true and the indices being scattered to are not unique then the semantics is implementation defined.